### DISCUSSION TITLE
Failed to start Docker - storage.journal.enabled

### DISCUSSION THREAD

**Discussion by i-B4se** [docker-library/mongo#653] [General]
Hey,

I run Rocketchat including MongoDB on my Unraid server. 
Since a few days I have problems to start MongoDB. Perhaps you can help me further.
I updated the two Dockers and since then MongoDB no longer starts with the following error message:

`Unrecognized option: storage.journal.enabled
try 'mongod --help' for more information`

This error message also occurs during a new installation.
Unfortunately I can not find a solution to start the database again.

Thank you in advance.

**Comment by LaurentGoderre** [comment]
Maybe you upgraded to a newer version where this option is no longer present? https://stackoverflow.com/a/76924946

**Comment by LaurentGoderre** [comment]
From Mongo: https://www.mongodb.com/docs/manual/core/journaling/#:~:text=Starting%20in%20MongoDB%206.1%2C%20journaling,%2D%2Dnojournal%20command%2Dline%20options.

> Starting in MongoDB 6.1, journaling is always enabled. As a result, MongoDB removes the storage.journal.enabled option and the corresponding --journal and --nojournal command-line options.
